lib/dash.coffee

```diff
@@ -55,7 +57,17 @@ plugin = module.exports =
       path = activeEditor.getPath()
       language = activeEditor.getGrammar().name
 
-    spawn('open', ['-g', @createLink(string, path, language)])
+    if platform == "win32"
+      # Exec is used because spawn escapes arguments that contain double-quotes
+      # and replaces them with backslashes. This interferes with the ability to
+      # properly create the child process in windows, since windows will barf
+      # on an ampersand that is not contained in double-quotes.
+      quotedLink = '"' + @createLink(string, path, language) + '"'
+      exec('cmd.exe /c start "" ' + quotedLink, null)
```

Comment:
I like https://github.com/sindresorhus/opn, it does look like a handy way to get around the issue, but without the ```-g``` option I see it as slightly limited without altering the library. Then again, including a whole library when it's only really needed for one line of code seems a bit overkill to me.